elements/nuxeo-app/nuxeo-expired-session.js

```diff
@@ -36,11 +36,10 @@ Polymer({
         height: var(--nuxeo-drawer-header-height);
         padding-left: 16px;
         position: fixed;
-        top: 0;
+        top: var(--nuxeo-app-top);
         left: 0;
         right: 0;
         z-index: 1000;
-        visibility: hidden;
         transform: translateY(-100%);
```

Comment:
We actually need to do some calculations here as well, otherwise, this banner will not be inside `nuxeo-app`, something like this (feel free to change it if you feel it can be more readable):
```suggestion
        transform: translateY(calc(-(100%) - (--var(--nuxeo-app-top, 0))));
```

Reply:
Same thing on `nuxeo-offline-banner`

Reply:
makes sense. I'm not sure what exactly this transform is doing. Do we need this property?

Reply:
we do, it is responsible for creating a slide animation from top to bottom, which is something that we've been doing for several versions now, so I think it is expected for us to maintain it.